general/virtuoso-features-matrix.ttl

```diff
@@ -0,0 +1,237 @@
+@prefix ns16:	<http://data.openlinksw.com/oplweb/product_feature/NetworkbasedQueryParallelization#> .
+@prefix ns17:	<http://data.openlinksw.com/oplweb/product_feature/ParallelizedSQLQueryExecution#> .
+@prefix ns18:	<http://data.openlinksw.com/oplweb/product_feature/RDFDataManagement#> .
+@prefix ns19:	<http://data.openlinksw.com/oplweb/product_feature/RDFMiddleware#> .
+@prefix ns20:	<http://data.openlinksw.com/oplweb/product_feature/RDFViewsoverExternalDataSources#> .
+@prefix ns21:	<http://data.openlinksw.com/oplweb/product_feature/RDFViewsoverNativeSQLDataSources#> .
+@prefix ns22:	<http://data.openlinksw.com/oplweb/product_feature/RDFawareClustering#> .
+@prefix ns23:	<http://data.openlinksw.com/oplweb/product_feature/RDFbasedLinkedDataDeployment#> .
+@prefix ns24:	<http://data.openlinksw.com/oplweb/product_feature/RunTimeHosting#> .
+@prefix ns25:	<http://data.openlinksw.com/oplweb/product_feature/SPARQLQueryLanguage#> .
+@prefix ns26:	<http://data.openlinksw.com/oplweb/product_feature/StorageEngine#> .
+@prefix ns27:	<http://data.openlinksw.com/oplweb/product_feature/SupportsBroadRangeofDataAccessStandards#> .
+@prefix ns28:	<http://data.openlinksw.com/oplweb/product_feature/WebDAVCompliantContentManager#> .
+@prefix ns29:	<http://data.openlinksw.com/oplweb/product_feature/WebServiceDeploymentPlatform#> .
+@prefix ns30:	<http://data.openlinksw.com/oplweb/product_feature/WebServicestoRDFLinkedData#> .
+@prefix dt:	<http://www.w3.org/2001/XMLSchema#> .
+@prefix ns32:	<http://data.openlinksw.com/oplweb/product_feature/AttributeBasedAccessControls#> .
+@prefix ns33:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease8x#> .
+@prefix ns34:	<http://data.openlinksw.com/oplweb/product_feature/CustomInferenceRules#> .
+@prefix ns35:	<http://data.openlinksw.com/oplweb/product_feature/EnhancedRelationMapping#> .
+@prefix ns36:	<http://data.openlinksw.com/oplweb/product_feature/column-wise-data-storage#> .
+@prefix ns37:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease70x#> .
+@prefix ns38:	<http://data.openlinksw.com/oplweb/product_feature/data-compression#> .
+@prefix ns39:	<http://data.openlinksw.com/oplweb/product_feature/elastic-cluster#> .
+@prefix ns40:	<http://data.openlinksw.com/oplweb/product_feature/new-extractor-cartridges#> .
+@prefix ns41:	<http://data.openlinksw.com/oplweb/product_feature/new-meta-cartridges#> .
+@prefix ns42:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoReleaseRowStoreClusterCloud64x#> .
+@prefix ns43:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoReleaseRowStoreSingle64x#> .
+@prefix ns44:	<http://data.openlinksw.com/oplweb/product_feature/vectored-execution#> .
+@prefix ns45:	<http://data.openlinksw.com/oplweb/product_feature/LinkedDataPublishing#> .
+@prefix ns46:	<http://data.openlinksw.com/oplweb/product_feature/virtuoso-server-high-performance#> .
+@prefix ns47:	<http://www.openlinksw.com/temp#> .
```

Comment:
This line should probably be removed, and anything using `ns47:` revised to another prefix/namespace.